template/report.html.tmpl

```diff
@@ -566,73 +630,120 @@
+          <td>{{ .postgres.After.CPUUsageTotal }}</td>
+          <td>{{ .postgres.Difference.CPUUsageTotal }}</td>
+          <td>{{ .fakerelay.Before.CPUUsageTotal }}</td>
+          <td>{{ .fakerelay.After.CPUUsageTotal }}</td>
+          <td>{{ .fakerelay.Difference.CPUUsageTotal }}</td>
+          {{- end }}
+        </tr>
+        {{ end }}
+      </table>
+    </section>
+    <section>
+      <h2 id="network">Network Traffic</h2>
+      <h3>Bytes In</h3>
+      <table>
+        <tr>
+          <th> </th>
+          <th>Total</th>
+          <th>Mean</th>
+        </tr>
+        {{ range .Data }}
         <tr>
           <th>{{ .Name }}</th>
           {{ with .TestResult.Metrics -}}
           <td>{{ .BytesIn.Total }}</td>
           <td>{{ .BytesIn.Mean }}</td>
           {{- end }}
         </tr>
+        {{ end }}
+      </table>
+    </section>
+    <section>
+      <h2 id="debug">Debugging</h2>
+
+      <h3>First Response from App</h3>
+      {{ range .Data }}
+      <p>{{ .Name }}</p>
+      <pre>{{ .TestResult.FirstAppResponse }}</pre>
       {{ end }}
-    </table>
 
-    <hr>
-    {{ range .Data }}
-      <details >
+      <hr>
+      <h3>First Request to Relay</h3>
+      <pre>{{ .FirstRequestHeaders }}</pre>
+      <div id="formatEnv">Loading...</div>
+      <div id="requestEnv" class="hidden">{{ .FirstRequestEnv }}</div>
+
+      <hr>
+      <h3>LoadGen Command</h3>
+      {{ range .Data }}
+      <p>{{ .Name }}</p>
+      {{ with .TestResult.LoadGenCommand -}}
+      <pre>{{ . }}</pre>
+      {{- end }}
+      {{- end }}
+
+      <hr>
+      {{ range .Data }}
+      <details>
         <summary>Percentile Table: <b>{{ .Name }}</b></summary>
         <div style="display: flex;">
           <pre class="hdr" data-name="{{ .Name }}">{{ .HDR }}</pre>
-          <button style="margin-top: 5px;height: 25px;margin-left: 10px;" onclick="copy('hdr', '{{ .Name }}')">Copy {{ .Name }} HDR</button>
+          <button style="margin-top: 5px;height: 25px;margin-left: 10px;" onclick="copy('hdr', '{{ .Name }}')">Copy
+            {{ .Name }} HDR</button>
         </div>
       </details>
       <details>
         <summary>Raw JSON: <b>{{ .Name }}</b></summary>
         <div style="display: flex;">
           <pre style="width: 400px;" class="json jsonFormat" data-name="{{ .Name }}">{{ .TestResultJSON }}</pre>
-          <button style="margin-top: 5px;height: 25px;margin-left: 10px;" onclick="copy('json', '{{ .Name }}')">Copy {{ .Name }} JSON</button>
+          <button style="margin-top: 5px;height: 25px;margin-left: 10px;" onclick="copy('json', '{{ .Name }}')">Copy
+            {{ .Name }} JSON</button>
         </div>
       </details>
-    {{ end }}
-
-    <hr>
-    <h3>First Response from App</h3>
-    {{ range .Data }}
-    <p>{{ .Name }}</p>
-    <pre>{{ .TestResult.FirstAppResponse }}</pre>
-    {{ end }}
-
-    <hr>
-    <h3>First Request to Relay</h3>
-    <details>
-      <summary>Request Headers</summary>
```

Comment:
Why no longer in details?

Reply:
Was an experiment. The double `<details>` was certainly not working for me, as I would find myself always having to click in two places instead of one.

Perhaps in "develop" mode it is okay not to have to click anything to see the debug section? We can/will probably remove it altogether when it comes to "publish" mode.